docs/source/modules/socrates.rst

```diff
@@ -0,0 +1,209 @@
+* Socrates radiation scheme requires ``mass-mixing`` ratios (mmr) for all quantities (e.g. CO2, water vapour etc). This contrasts with RRTM, which wants ``volume-mixing`` ratios (vmr).
+
+
+Namelist options
+---------------- 
+
+The Socrates namelist ``socrates_rad_nml`` can be found at ``src/atmos_param/socrates/interface/socrates_config_mod.f90``.
+
+Radiation options
+^^^^^^^^^^^^^^^^^
+
+Here are some options to set incoming radiation:
+
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+| Name                       | Default  | Description                                                                             |
++============================+==========+=========================================================================================+
+|``solday``                  | 0        | If >0, do perpetual run corresponding to day of the year = solday in [0, days per year] |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``do_rad_time_avg``         | True     | Average ``coszen`` for shortwave radiation over ``dt_rad``                              |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``equinox_day``             | 0.75     | fraction of the year defining NH autumn equinox in [0, 1]                               |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``stellar_constant``        | 1368.22  | :math:`Wm^{-2}`, solar constant, consistent with RRTM default                           |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``tidally_locked``          | False    | Tidally locked or not                                                                   |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``frierson_solar_rad``      | False    | Options for annual-mean incoming solar, as prescribed in Frierson's grey scheme         |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``del_sol``                 | 1.4      | Latitudinal variation of shortwave radiation, as prescribed in Frierson's grey scheme   |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``del_sw``                  | 0        |                                                                                         |
```

Comment:
This is another parameter that controls the latitudinal variation of shortwave radiation in Frierson.